sdk/highlight-py/highlight_io/sdk.py

```diff
@@ -44,6 +46,7 @@ class H(object):
     OTLP_HTTP = "https://otel.highlight.io:4318"
     _instance: "H" = None
     _logging_instrumented = False
+    _context_map = LRUCache(1000)
```

Comment:
a comment here would be useful for the future why we use an LRU-expiring map here